plugins/hypervisors/kvm/src/com/cloud/ha/KVMInvestigator.java

```diff
@@ -67,6 +73,10 @@ public Status isAgentAlive(Host agent) {
             return null;
         }
 
+        if (haManager.isHAEligible(agent)) {
```

Comment:
@koushik-das FYI - injections here for VM-HA Host-HA coordination, the eligibility checks several things -- including that the feature is explicitly enabled for a host.